resources/js/calendar-widget.js

```diff
@@ -62,7 +64,7 @@ export default function calendarWidget({
                 };
             }
 
-            if (hasDateSelectContextMenu) {
+            if (hasDateSelectContextMenu && !dateSelectEnabled) {
```

Comment:
Please refactor this into an else if branch. Prioritize custom event -> contextMenu, so first check dateSelectEnabled and after that context menu.

Reply:
See my reply on the likewise logic.

Reply:
Thanks, I refactored it a little bit, since to match the other events.